app/design/frontend/UnitedArtsEducation/unitednow/web/css/source/_extend.less

```diff
@@ -14,3 +14,19 @@
         border-color: @active__color;
     }
 }
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {
+    .page-layout-1column{
+        .column.main{
+            .product{
+                &.media{
+                    width:45%;
+                }
+            }
+
+            .product-info-main{
+                width:50%;
+            }
```

Comment:
With these columns being 45% & 50%, respectively, there is now a 5% gap, instead of 3%. Did you intend to make the gap this much bigger?

Reply:
I just thought it looked best like this, but I can change it to 45/52